src/main/java/mekanism/common/CommonPlayerTickHandler.java

```diff
@@ -179,21 +182,24 @@ public static boolean handleJetpackMotion(Player player, JetpackMode mode, Boole
         return true;
     }
 
-    private static boolean isJetpackOn(Player player, ItemStack chest) {
+    private static ItemStack getJetpackIfOn(ItemStack chest, Player player) {
```

Comment:
No reason to reorder the parameters here.

Reply:
Oh ups.